heartbeat.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/moira-alert/cache/filter"
+)
+
+func heartbeat(db *filter.DbConnector, terminate chan bool, wg *sync.WaitGroup) {
+	defer wg.Done()
+	count := filter.TotalMetricsReceived.Count()
+	for {
+		select {
+		case <-terminate:
+			return
+		case <-time.After(time.Second * 5):
```

Comment:
This is not a check, this is heartbeat sending.
5 sec should be enough for any case and does't hit a performance I guess.
If we introduce new settings, user can misconfigure it.